quill-async/src/main/scala/io/getquill/sources/async/MysqlAsyncSource.scala

```diff
@@ -0,0 +1,18 @@
+package io.getquill.sources.async
+
+import com.github.mauricio.async.db.mysql.MySQLQueryResult
+import com.github.mauricio.async.db.{ QueryResult => DBQueryResult, Connection }
+
+import io.getquill.naming.NamingStrategy
+import io.getquill.sources.sql.idiom.SqlIdiom
+
+class MysqlAsyncSource[D <: SqlIdiom, N <: NamingStrategy, C <: Connection](config: AsyncSourceConfig[D, N, C]) extends AsyncSource[D, N, C](config) {
+
+  override protected def extractGenerated(generated: Option[String])(result: DBQueryResult): Long = {
```

Comment:
This  is doing  `extractActionResult`, not `extractGenerated` actually, same to `PostgresAsyncSource`